config/example/game_options.txt

```diff
@@ -11,7 +11,7 @@ REVIVAL_POD_PLANTS 1
 REVIVAL_CLONING 1
 
 ## Amount of time (in hundredths of seconds) for which a brain retains the "spark of life" after the person's death (set to -1 for infinite)
-REVIVAL_BRAIN_LIFE -1
+REVIVAL_BRAIN_LIFE 60000
```

Comment:
Uhhhh. what. Since when was this ever a thing?

Reply:
I don't know WHY this is there, or HOW LONG it has been there.
(I can't answer those questions because nsscyberiad.net, which hosted the gitlab install used to track versioning on the old/current server, is down)

All I can tell you for a fact is that it IS there.
If it shouldn't be we can take it out, but this example kinda demonstrates why this PR was necessary... there are lots of things that were in the production config which maybe shouldn't have been.

I have been updating the config (on production test server, as I review it, but there are gonna be cases like this where I don't see the setting as remarkable, but someone else does.